sdks/python/apache_beam/pipeline.py

```diff
@@ -152,6 +153,12 @@ def __init__(self, runner=None, options=None, argv=None):
       raise ValueError(
           'Pipeline has validations errors: \n' + '\n'.join(errors))
 
+    # set default experiments for portable runner
+    # (needs to occur prior to pipeline construction)
+    if self._options.view_as(StandardOptions).runner == 'PortableRunner':
+      self._options.view_as(DebugOptions).experiments = (
+            self._options.view_as(DebugOptions).experiments or ['beam_fn_api'])
```

Comment:
We should check and append`beam_fn_api` to experiments.

Reply:
The idea was to only touch experiments when the user did not set it. Do you think we should just always add `beam_fn_api` instead?

Reply:
I think it should be ok to add beam_fn_api always as without it the pipeline will fail. 
And the behavior will be inconsistent where when no experiment is provided then the pipeline works but when when experiments is provided without explicit beam_fn_api, the pipeline fails.

Reply:
done!